site/en/docs/extensions/reference/scripting/index.md

```diff
@@ -215,18 +231,49 @@ async function addIframe() {
   return iframe.contentWindow.document.title;
 }
 
-const tabId = getTabId();
-chrome.scripting.executeScript(
-    {
-      target: {tabId: tabId, allFrames: true},
-      func: addIframe,
-    },
-    (injectionResults) => {
-      for (const frameResult of injectionResults)
-        console.log('Iframe Title: ' + frameResult.result);
-    });
+chrome.scripting
+  .executeScript({
+    target: {tabId: getTabId(), allFrames: true},
+    func: addIframe,
+  })
+  .then(injectionResults => {
+    for (const frameResult of injectionResults) {
+      const {frameId, result} = frameResult;
+      console.log(`Frame ${frameId} result:`, result);
+    }
+  });
+```
+
+## Examples
```

Comment:
It seems like all the preceding code examples were also examples - what determines when we put one here vs there?

Reply:
The short answer is recency. I began adding "Examples" headers to other API reference pages around a year ago in an attempt to highlight common patterns and ways that these APIs are used. The existing examples on this page predate that work and while authoring this PR I felt that normalizing the organization of the samples would be too much scope creep for this PR.

Reply:
I'd like to try to land this PR as is and follow up with additional cleanup.

Reply:
It looks like 'Usage' is more like 'this API 101' while examples are use case specific. That's a valid distinction, but I think it will be lost on most readers. Labeling it that way might create unreasonable expectations about how much we can provide in this section. I think it's also inevitable that there will be examples where the label they should be under is not clear. At that point the distinction will become meaningless. 

I would simplify and come up with one heading to rule them all.